.github/workflows/rusty.yaml

```
@@ -0,0 +1,145 @@
+    runs-on: ubuntu-latest
+    
+    services:
+      # Sybase ASE Docker コンテナ
+      sybase:
+        image: datagrip/sybase:16.0
+        env:
+          SYBASE_DB: testdb
+          SYBASE_USER: testuser
+          SYBASE_PASSWORD: testpass
+        ports:
+          - 5000:5000
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)